src/users.js

```diff
@@ -27,10 +27,14 @@ function Users(clientApi) {
   this.api = clientApi;
 }
 
-Users.prototype.usersList = function (contextid, callback) {
+/**
+ * Gets a list of users that belong to the context with the given ID, or all users if none provided.
+ * @param {string} contextid - The Context Id
```

Comment:
Core could be regenerated again (or find and replace), the `Id` was changed to `ID` in all places.